contents/handbook/why-does-posthog-exist.md

```diff
@@ -6,24 +6,21 @@ showTitle: true
 
 ## Our mission
 
-To help engineers build better products.
+Equip every developer to build successful products
 
 ## Our strategy
 
-### Provide every tool needed for evaluating feature success
+### Provide every tool needed to build successful products
 
-We want engineers to engineer their products, not their data, and to not have to hire lots of product managers to pull together context from lots of tools.
+Building a successful product is hard. It gets harder when you don't understand how your customers are using your product.
 
-By integrating all these tools we can make this easy – no integration needed, no extra vendors, no extra JavaScript, no sending data to extra third parties, and workflows to guide engineers through feature development.
+PostHog gives you all the tools you need to understand how your users use your product. All with one simple snippet.
```

Comment:
Just a fly-by, but we may want to change to

```suggestion
PostHog includes all the tools you need to understand how people use your product. All with one simple snippet.
```

"how your users use your" is just a bit of a mouthful